src/main/java/org/openstreetmap/atlas/checks/maproulette/serializer/ChallengeDeserializer.java

```diff
@@ -45,8 +45,8 @@ public Challenge deserialize(final JsonElement json, final Type typeOfT,
         ChallengePriority priority;
         try
         {
-            priority = ChallengePriority.valueOf(
-                    this.getStringValue(challengeObject, Challenge.KEY_DEFAULT_PRIORITY, ""));
+            priority = ChallengePriority.valueOf(this.getStringValue(challengeObject,
+                    Challenge.KEY_DEFAULT_PRIORITY, Challenge.KEY_DEFAULT_PRIORITY_VALUE + ""));
```

Comment:
I'd suggest using `String.valueOf` rather than concatenation with empty string. Check [this](https://stackoverflow.com/a/7752401/853005) out.